### PR TITLE
Rename statistics

### PR DESCRIPTION
**Pull request by AlexTheLion123**
# Goal

- Fix incorrectly named statistics, especially statistics that mistakenly referred to `trades` when they should've been referring to `positions`
- Standardize variable names to be more clear and also refer to the past e.g. `won_positions` - not `winning_positions` and not just `won`
- Clean genenral code for `calculate_summary_statistics()` method

### CODE REVIEW

**Review by kenkunz — COMMENTED** [review]
See comment on renaming the class instead of the accessors.

In addition… tests need to be passing before this is merged (I'm probably not the best person to help with that), and we need to agree on the simplest approach to handling the name changes in consumers.

File: tradeexecutor/analysis/trade_analyser.py
[the remark this answers is not in the document]
Comment (AlexTheLion123) [reply tradeexecutor/analysis/trade_analyser.py:1]: Thanks Ken, noted, will have a look at this again today

[the remark this answers is not in the document]
Comment (AlexTheLion123) [reply tradeexecutor/analysis/trade_analyser.py:1]: Yeah this is related to [previous conversation](https://discord.com/channels/888448459168948284/1089966098134356048/1089966102412529665) we had

**Comment by AlexTheLion123** [comment]
After conferring with @miohtama, it looks like renaming stats here is unnecessary for now, have created ticket #268 to do it at a later stage. Just adding comments and changing text descriptions for now. Only frontend needs to be updated

**Pull request closed without merging**